src/ui/components/CreateIdentifier/CreateIdentifier.tsx

```diff
@@ -22,7 +22,7 @@ const CreateIdentifier = ({
   const initialState = {
     identifierCreationStage: 0,
     displayNameValue: "",
-    selectedIdentifierType: 0,
+    selectedIdentifierType: 1,
```

Comment:
We shouldn't need this now, since you can't select between the 2 of them (or if we can, it should be removed)